src/client/nodejs/test/TcpEventBusBridgeEchoServer.java

```diff
@@ -0,0 +1,28 @@
+package io.vertx.ext.eventbus.bridge.tcp;
```

Comment:
it would be better to clearly separate the main source of the client from the test sources